tests/layer_manager.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { AnyModel } from '@anywidget/types';
+import '../js/layer_manager';
+import { default as layerManagerRender, LayerManager, LayerManagerModel } from '../js/layer_manager';
+import { FakeAnyModel } from './fake_anywidget';
+
+describe('<layer-manager>', () => {
+    let layerManager: LayerManager;
+
+    async function makeManager(model: AnyModel<LayerManagerModel>) {
+        const container = document.createElement('div');
+        layerManagerRender.render({
+            model, el: container, experimental: {
+                invoke: () => new Promise(() => [model, []]),
+            }
+        });
+        const element = container.firstElementChild as LayerManager;
+        document.body.appendChild(element);
+        await element.updateComplete;
+        return element;
+    }
+
+    beforeEach(async () => {
+        layerManager = await makeManager(new FakeAnyModel<LayerManagerModel>({
+            visible: true,
+            children: [],
```

Comment:
What about adding some children to the LayerManager and verifying they are added?

Reply:
Ah, unfortunately, this is a bit complicated, as it requires adding deeper mocks to the `FakeAnywidget` that can handle widget production. I think it's achievable but I don't think I understand it well enough to be able to get it done in a reasonable amount of time. I think instead we can try to run tests for `utils.ts` since that's where the `updateChildren` is called, but I'll leave that as a TODO.